manifests/u/UweSieber/USBDLM/5.6.1.0/UweSieber.USBDLM.installer.yaml

```
@@ -0,0 +1,18 @@
+# Automatically updated by the winget bot at 2024/Jun/04
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: UweSieber.USBDLM
+PackageVersion: 5.6.1.0
+InstallerType: wix
+Scope: machine
+UpgradeBehavior: install
+ReleaseDate: 2023-01-22
+AppsAndFeaturesEntries:
+- DisplayVersion: 5.6.0.0

```

```suggestion
- DisplayVersion: 5.6.1.0
```
This is purely guessing, pelase forgive me if incorrect.